src/Neo/SmartContract/Native/ContractEventAttribute.cs

```diff
@@ -0,0 +1,165 @@
+// Copyright (C) 2015-2024 The Neo Project.
```

Comment:
These files and simplification could be in another PR, right?
If you want to merge this first.